docs/plus/cronjob.md

```diff
@@ -0,0 +1,93 @@
+# CronJob
+
+CronJob resource is responsible for creating recurring [Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/job/). The job recurrence is determined by a [Schedule]() provided. You can also specify a custom schedule by passing a [cron pattern](https://en.wikipedia.org/wiki/Cron) to Schedule's cron method.
+
```

Comment:
Add a link to the future API Reference page, like here:

https://github.com/cdk8s-team/cdk8s/blob/ade2e3de0d8b7193e09804f5fbf37c5f6bada7d5/docs/plus/config-map.md?plain=1#L6-L7

Reply:
Will add it.